sdk-core/src/main/java/care/data4life/sdk/RecordService.kt

```diff
@@ -965,10 +965,14 @@ class RecordService internal constructor(
                         )
                     newAttachment.id == null -> validAttachments.add(newAttachment)
                     else -> {
-                        val oldAttachment = oldAttachments[newAttachment.id]
-                            ?: throw DataValidationException.IdUsageViolation(
-                                "Valid Attachment.id expected"
-                            )
+                        val oldAttachment = oldAttachments.getOrElse(
```

Comment:
it was somehow easier for me to understand before 😅 
At least now i have it clear either way

Reply:
I would prefer the previous version as it also fits to how we use it in other places